example_zoo/tools/templates/submit.sh

```diff
@@ -36,4 +36,4 @@ gcloud ai-platform jobs submit training $JOB_NAME \
     --python-version 2.7 \
```

Comment:
Is Python 2.7 a requirement? Can we use 3.5 instead?

Reply:
Good point - I'll try to add it.